### PR TITLE
New presense sensor. Model YXZBRB58

### PR DESCRIPTION
**Pull request by kirovilya**
https://aliexpress.ru/item/1005005744570857.html

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -5381,6 +5381,51 @@ const definitions: Definition[] = [
...
+                .withDescription('Maximum range').withUnit('m'),
+            e.numeric('detection_delay', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(0.1)
+                .withDescription('Detection delay').withUnit('s'),
+            e.numeric('fading_time', ea.STATE_SET).withValueMin(0).withValueMax(1500).withValueStep(1)
+                .withDescription('Fading time').withUnit('s'),
+            e.enum('radar_scene', ea.STATE_SET, ['default', 'bathroom', 'bedroom', 'sleeping'])
+                .withDescription('Presets for sensitivity for presence and movement'),
+        ],
+        meta: {
+            tuyaSendCommand: 'dataRequest',
```
Comment (Koenkk) [inline src/devices/tuya.ts:5408]: The default is `dataRequest` so this shouldn't be needed (please test)

Comment (kirovilya) [reply src/devices/tuya.ts]: yes, working

**Comment by blakadder** [comment]
Seems minimum and maximum range dp's are reversed
image

**Comment by fliespl** [comment]
@kirovilya I have some strange issues with this converted.

Presets set values that are out of scope for specific items. I.e. bathroom sets 60s detection time, while detection time (based on your investigation) should be within 0-10. 

Something is really off here.

**Comment by blakadder** [comment]
> @kirovilya I have some strange issues with this converted.
> 
> Presets set values that are out of scope for specific items. I.e. bathroom sets 60s detection time, while detection time (based on your investigation) should be within 0-10. 
> 
> Something is really off here.

I PRd a fix yesterday for swapped dp's

**Pull request merged**